kiwi/boot/image/dracut.py

```diff
@@ -46,7 +46,9 @@ def create_initrd(self, mbrid=None):
         if self.is_prepared():
             log.info('Creating generic dracut initrd archive')
             kernel_info = Kernel(self.boot_root_directory)
-            kernel_version = kernel_info.get_kernel().version
+            kernel_details = kernel_info.get_kernel(
+                raise_on_not_found=True
+            )
```

Comment:
That usually _has_ to be one line, unless multi-params. Otherwise not really "pythonic" code.